### PR TITLE
Model documentation simon

### PR DESCRIPTION
**Pull request by edmondchuc**
@dr-shorthair I've made some updates to address your PR in https://github.com/ternaustralia/linkeddata-site/pull/13.

### CODE REVIEW

File: tern-ontology/conceptual-information-model.md [hunk 1]
```diff
@@ -217,22 +212,22 @@ It is later sent to the local state/territory herbarium for identification.
...
 Sites may contain nested 'sites' using the sub-sample relationship. 
 The site-type is indicated using a value from a controlled vocabulary. 
-Common site-types may include `plot`, `transect`, `quadrat`. 
-Other features identified within a Site are also represented using sub-sample relationships.
+Common site-types may include `plot`, `transect`, `quadrat`. TERN manages a list of [site types](http://linked.data.gov.au/def/tern-cv/74aa68d3-28fd-468d-8ff5-7e791d9f7159) as a SKOS controlled vocabulary.
+Other features and samples defined within a site are connected back to the site through the `is sample of` relationship.
 
 A Site may itself be a kind of Sample, as it may be deemed to be representative of a
-- fiat area such as a parcel, a local government area, state or territory
+- flat area such as a parcel, a local government area, state or territory
```
Comment (dr-shorthair) [inline tern-ontology/conceptual-information-model.md]: 'fiat' is correct. It means that the thing is defined by drawing lines on a map, in contrast to by some natural phenomena.

Comment (edmondchuc) [reply tern-ontology/conceptual-information-model.md]: Haha oops, sorry, I'll fix that!

File: tern-ontology/conceptual-information-model.md [hunk 2]
```diff
@@ -248,9 +243,9 @@ Studies on the collected data provide us with an understanding of why and how th
 
 ## Spatial geometries
 
-Sites, site visits, samplings and observations can use the TERN Location Alignment vocabulary, which is a minimal ontology to tie together the GeoSPARQL vocabulary and the W3C Basic Geo WGS84 vocabulary. 
-TERN Location Alignment also defines a specialised Point class with properties to express the depth and elevation. 
-LineString and Polygon are also specialised to denote that a WKT literal is required.
+Sites, site visits, samplings and observations can use the TERN Location Alignment vocabulary, which is a minimal ontology to tie together the GeoSPARQL vocabulary and the W3C Basic Geo WGS84 vocabulary. See https://w3id.org/tern/ontologies/loc.
```
Comment (dr-shorthair) [inline tern-ontology/conceptual-information-model.md:246]: My general recommendation is to use GeoSPARQL and WKT as much as possible. LOC gives the alternative of addresses or locations denoted by some text string, but in practice I think we will usually have coordinate-based geometries?

**Review by dr-shorthair — COMMENTED** [review]
A couple of comments below. 'fiat' needs to stay as is. The comment about geometry representations does not affect this documentation at this time, but is food-for-thought later on.

File: tern-ontology/conceptual-information-model.md (hunk 2 above)
Comment (edmondchuc) [reply tern-ontology/conceptual-information-model.md:246]: > LOC gives the alternative of addresses or locations denoted by some text string

[`tern-loc:Point`](https://w3id.org/tern/ontologies/loc/Point) basically defines a few constraints to require `wgs:lat`, `wgs:long` and `geosparql:asWKT`. 

Not sure if we are looking at the same thing. I don't see any references to addresses or locations denoted by text string.

[These](https://linkeddata-dev.tern.org.au/viewers/tern-loc-ontology) are the current geometries defined in `tern-loc` for use with the TERN Ontology.

> but in practice I think we will usually have coordinate-based geometries?

Currently the 3 geometries defined in `tern-loc` all require a `wktLiteral` to represent the geometry.

**Comment by edmondchuc** [comment]
Thanks @dr-shorthair, I've resolved the comment relating to `fiat` and I've left a reply to the other comment.

File: tern-ontology/conceptual-information-model.md (hunk 2 above)
Comment (dr-shorthair) [reply tern-ontology/conceptual-information-model.md:246]: I was incorrect. I had not looked property at tern-loc - I had conflated it with https://www.w3.org/ns/locn

**Pull request merged**